src/test/java/Drivers/ChromeDriverManager.java

```diff
@@ -0,0 +1,22 @@
+package Drivers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class ChromeDriverManager {
+    public WebDriver getDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/test/java/resources/chromedriver.exe");
+
+        ChromeOptions options = new ChromeOptions();
+
+        options.addArguments("--start-maximized");
+        options.addArguments("--disable-extensions");
+        options.addArguments("--disable-infobars");
+        options.addArguments("--disable-notifications");
+        options.addArguments("--disable-popup-blocking");
+        options.addArguments("--incognito");
```

Comment:
Do we need all the flags for our test cases?

Reply:
@fnfldx To be discussed tomorrow.

Reply:
@fnfldx All flags removed.